x-pack/plugins/security_solution/public/resolver/store/data/reducer.ts

```diff
@@ -153,6 +157,33 @@ export const dataReducer: Reducer<DataState, ResolverAction> = (state = initialS
       // the action is stale, ignore it
       return state;
     }
+  } else if (action.type === 'appRequestedCurrentRelatedEventData') {
+    const nextState: DataState = {
+      ...state,
+      currentRelatedEvent: {
+        loading: true,
+        data: null,
+      },
+    };
+    return nextState;
+  } else if (action.type === 'serverReturnedCurrentRelatedEventData') {
+    const nextState: DataState = {
+      ...state,
+      currentRelatedEvent: {
+        loading: false,
+        data: action.payload,
+      },
+    };
+    return nextState;
+  } else if (action.type === 'serverFailedToReturnCurrentRelatedEventData') {
```

Comment:
❔ Should this reduce differently than the initial state? They don't seem to be distinguishable based on state. (both are loading: false, data: null) Is that OK?

Reply:
Yea, they should be the same. I can probably have a default object or something that I use to relay that intent. I can do that in a follow up PR